src/cli/commands/start/start.ts

```diff
@@ -298,4 +303,15 @@ export async function start(options: SWACLIConfig) {
     .catch((err) => {
       logger.error(err.message, true);
     });
+
+  const end = new Date().getTime();
+
+  collectTelemetryEvent(
+    TELEMETRY_EVENTS.Start,
+    {
+      apiRuntime: swaConfigFileContent?.platform.apiRuntime!,
+      duration: (end - start).toLocaleString(),
+    },
+    { appRuntime: nodeMajorVersion }
```

Comment:
how the metric will look like for tappRuntime ?

Reply:
It'll look like node 16